common/src/main/java/org/bonitasoft/console/common/server/utils/PlatformManagementUtils.java

```diff
@@ -71,9 +71,20 @@ private void retrievePlatformConfiguration(PlatformAPI platformAPI) throws IOExc
 
     public void initializePlatformConfiguration() throws BonitaException, IOException {
         PlatformSession platformSession = platformLogin();
-        PlatformAPI platformAPI = PlatformAPIAccessor.getPlatformAPI(platformSession);
+        PlatformAPI platformAPI = getPlatformAPI(platformSession);
         retrievePlatformConfiguration(platformAPI);
         retrieveTenantsConfiguration(platformAPI);
         platformLogout(platformSession);
     }
+
+    public void updateConfigurationFile(long tenantId, String file, byte[] content) throws IOException, BonitaException {
+        PlatformSession platformSession = platformLogin();
+        PlatformAPI platformAPI = getPlatformAPI(platformSession);
+        platformAPI.updateClientTenantConfigurationFile(tenantId, file, content);
+        platformLogout(platformSession);
+    }
+
+    PlatformAPI getPlatformAPI(PlatformSession platformSession) throws BonitaHomeNotSetException, ServerAPIException, UnknownAPITypeException {
```

Comment:
Removing BonitaHomeNotSetException from the throws clause of getServerAPI() would be an API break?